chrome-launcher/chrome-launcher.ts

```diff
@@ -310,3 +367,9 @@ export class Launcher {
     });
   }
 };
+
+const manager = new LaunchManager();
```

Comment:
We should avoid a singleton if possible. Won't this cause issues e.g. launching two instances of Chrome from a single node process and that `attempts` won't reset between runs?